collect_app/src/main/java/org/odk/collect/android/preferences/AdminPreferencesFragment.java

```diff
@@ -36,14 +35,10 @@ public void onCreate(final Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         PreferenceManager prefMgr = getPreferenceManager();
         prefMgr.setSharedPreferencesName(ADMIN_PREFERENCES);
-        prefMgr.setSharedPreferencesMode(MODE_WORLD_READABLE);
+        //prefMgr.setSharedPreferencesMode(MODE_WORLD_READABLE);
```

Comment:
I don't think this change is related to your issue so it shouldn't be included. If you'd like to either remove this or consider an alternative because it is dangerous, please file an issue so we can discuss.